pyiceberg/utils/schema_conversion.py

```diff
@@ -68,8 +68,9 @@
 
 LOGICAL_FIELD_TYPE_MAPPING: Dict[Tuple[str, str], PrimitiveType] = {
     ("date", "int"): DateType(),
+    ("time-millis", "int"): TimeType(),
     ("time-micros", "long"): TimeType(),
-    ("timestamp-millis", "int"): TimestampType(),
+    ("timestamp-millis", "long"): TimestampType(),
```

Comment:
good catch! heres the corresponding type conversion for `timestamp-millis` in java https://github.com/apache/iceberg/blob/d3d5662aee19dd1799fd12740c64eadcb7f8da8b/core/src/main/java/org/apache/iceberg/avro/GenericAvroReader.java#L184-L187

I see `time-micros` but I dont see `time-millis` in here though

Reply:
Interesting. I was just looking into the Avro Spec for logicalTypes and not in the iceberg Java Code.

Reply:
This was intentional since Iceberg does not support millis, just micros, and nanos in V3+